app/image/image-controller.js

```diff
@@ -8,21 +8,26 @@
  * Controller of the docker-registry-frontend
  */
 angular.module('image-controller', ['registry-services', 'app-mode-services'])
-  .controller('ImageController', ['$scope', '$route', '$routeParams', '$location', '$log', '$filter', 'Image', 'Ancestry', 'AppMode',
-  function($scope, $route, $routeParams, $location, $log, $filter, Image, Ancestry, AppMode){
-    $scope.imageId = $route.current.params.imageId;
-    $scope.imageDetails = Image.query( {imageId: $scope.imageId} );
-    $scope.imageAncestry = Ancestry.query( {imageId: $scope.imageId} );
+  .controller('ImageController', ['$scope', '$route', '$routeParams', '$location', '$log', '$filter','Manifest', 'Image', 'Ancestry', 'AppMode',
+  function($scope, $route, $routeParams, $location, $log, $filter, Manifest, Image, Ancestry, AppMode){
+    
+
+    // Old code (v1 specs)
+    // $scope.imageId = $route.current.params.imageId;
+    // $scope.imageDetails = Image.query( {imageId: $scope.imageId} );
+    // $scope.imageAncestry = Ancestry.query( {imageId: $scope.imageId} );
     $scope.appMode = AppMode.query();
+    $scope.imageDetails = Manifest.query({repoUser: $scope.repositoryUser, repoName: $scope.repositoryName, tagName: $scope.tagName});
     /**
      * Calculates the total download size for the image based on
      * it's ancestry.
      */
+    /*
```

Comment:
If you effectively delete code by commenting it out, please simply remove it instead of keeping it. I left it there to pick it up when I have time. Now that you've implemented it, there's now need to keep the old code around unless there's something you want to implement in the future.